### PR TITLE
feat: add created_by and status columns to change_request_schedule

### PR DESCRIPTION
**Pull request by andreas-unleash**
Adds 2 columns to change_request_schedule

### CODE REVIEW

File: src/migrations/20231030091931-add-created-by-and-status-change-request-schedule.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+exports.up = function (db, cb) {
+  db.runSql(`
+        ALTER TABLE change_request_schedule
+            ADD COLUMN IF NOT EXISTS created_by integer REFERENCES users(id);
+
+        ALTER TABLE change_request_schedule
+            ADD COLUMN IF NOT EXISTS status text default 'pending';
```
Comment (thomasheartman) [inline src/migrations/20231030091931-add-created-by-and-status-change-request-schedule.js:9]: Would we wanna use enum types for this or is text preferable? How do we do it in other places?

File: src/migrations/20231030091931-add-created-by-and-status-change-request-schedule.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+exports.up = function (db, cb) {
+  db.runSql(`
+        ALTER TABLE change_request_schedule
+            ADD COLUMN IF NOT EXISTS created_by integer REFERENCES users(id);
```
Comment (thomasheartman) [inline src/migrations/20231030091931-add-created-by-and-status-change-request-schedule.js:6]: Would `scheduled_by` be a better column name than `created_by` in this instance?

```suggestion
            ADD COLUMN IF NOT EXISTS scheduled_by integer REFERENCES users(id);
```

Comment (andreas-unleash) [reply src/migrations/20231030091931-add-created-by-and-status-change-request-schedule.js:6]: either way is fine with me - row was created by someone is what matters

Comment (thomasheartman) [reply src/migrations/20231030091931-add-created-by-and-status-change-request-schedule.js:6]: Got it! I'd lean towards "scheduled_by", but I'll let it be up to you in the end ☺️

File: src/migrations/20231030091931-add-created-by-and-status-change-request-schedule.js (hunk 1 above)
Comment (andreas-unleash) [reply src/migrations/20231030091931-add-created-by-and-status-change-request-schedule.js:9]: makes sense

Comment (thomasheartman) [reply src/migrations/20231030091931-add-created-by-and-status-change-request-schedule.js]: Sure. I think the most important bit is to stay consistent with how we do it other places, so I'd check for that first, and do what we do there. If we have no precedents for this, I think an enum makes sense. But don't trust me blindly on that 😅

**Review by thomasheartman — APPROVED** [review]
I'll approve this 😄  It's a fine change, and I'm expecting you to handle the comments in a sensible way. You've seen them, so it's up to you what to do with them. Thanks and good work 🙌🏼

File: src/migrations/20231030091931-add-created-by-and-status-change-request-schedule.js (hunk 1 above)
Comment (ivarconr) [reply src/migrations/20231030091931-add-created-by-and-status-change-request-schedule.js]: https://github.com/Unleash/unleash/commit/6875daf15768d026a53bbc1916fb5c4219fe5680#commitcomment-131584329

**Pull request merged**